backend/hitas/models/apartment.py

```diff
@@ -148,3 +156,24 @@ class ApartmentConstructionPriceImprovement(HitasImprovement):
     apartment = models.ForeignKey("Apartment", on_delete=models.CASCADE, related_name="construction_price_improvements")
 
     depreciation_percentage = EnumField(DepreciationPercentage, default=DepreciationPercentage.TEN)
+
+
+class ApartmentMaxPriceCalculation(models.Model):
```

Comment:
The model still has the 'old' `max` instead of `maximum`, up to you if you want to change _everything_ 😄 
(also tests, filenames and some comments have `max`)

Reply:
I know, I'll keep it as is :) IMO `max` is fine in the code but it's nicer to expose the word publicly in urls etc with the full form